From: David Steele <steele@debian.org>
Date: Fri, 30 Sep 2016 21:21:35 -0400
Subject: Fix the help message

The help message had some escapes normally filled in by the bash build
script. Remove them.
---
 main.go | 14 ++------------
 1 file changed, 2 insertions(+), 12 deletions(-)

diff --git a/main.go b/main.go
index 9301ae1..1ce8572 100644
--- a/main.go
+++ b/main.go
@@ -114,18 +114,8 @@ func changePassword(args *argContainer) {
 // printVersion prints a version string like this:
 // gocryptfs v0.12-36-ge021b9d-dirty; go-fuse a4c968c; 2016-07-03 go1.6.2
 func printVersion() {
-	humanTime := "0000-00-00"
-	if i, _ := strconv.ParseInt(BuildTime, 10, 64); i > 0 {
-		t := time.Unix(i, 0).UTC()
-		humanTime = fmt.Sprintf("%d-%02d-%02d", t.Year(), t.Month(), t.Day())
-	}
-	buildFlags := ""
-	if stupidgcm.BuiltWithoutOpenssl {
-		buildFlags = " without_openssl"
-	}
-	built := fmt.Sprintf("%s %s", humanTime, runtime.Version())
-	fmt.Printf("%s %s%s; go-fuse %s; %s\n",
-		tlog.ProgramName, GitVersion, buildFlags, GitVersionFuse, built)
+	fmt.Printf("%s\n",
+		tlog.ProgramName)
 }
 
 func main() {
